src/svgcontext.ts

```diff
@@ -34,6 +34,10 @@ const attrNamesToIgnoreMap: { [nodeName: string]: Attributes } = {
 /** Create the SVG in the SVG namespace. */
 const SVG_NS = 'http://www.w3.org/2000/svg';
 
+function round3(x: number): string {
+  return (Math.round(x * 1000) / 1000).toString();
```

Comment:
All comments online about incorrect rounding in this case are irrelevant because we don't even really care about the third decimal place (but that's why it's better to keep it than try for 2 places).